src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SqlDataRecord.cs

```diff
@@ -54,10 +54,24 @@ public virtual string GetDataTypeName(int ordinal)
 #if NET
         [return: DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.PublicFields)]
 #endif
-        public virtual Type GetFieldType(int ordinal) => GetFieldTypeFrameworkSpecific(ordinal);
+        public virtual Type GetFieldType(int ordinal)
+        {
+            SqlMetaData md = GetSqlMetaData(ordinal);
+
+#if NETFRAMEWORK
+            if (md.SqlDbType == SqlDbType.Udt)
```

Comment:
Since this check only applies to netframework and will return, let's avoid the braces for netcore

```c#
#if NETFRAMEWORK
if (md.SqlDbType == SqlDbType.Udt)
{
    return md.Type;
}
#endif

return MetaType.GetMetaTypeFromSqlDbType(md.SqlDbType, false).ClassType;

Reply:
Also tiny request, can we get an argument label for the `false` here? :)